### DISCUSSION TITLE
Grafana 9.1.0 installs xorg-x11-server-utils, what is using it?

### DISCUSSION THREAD

**Discussion by bts-dido** [grafana/grafana#62993] [General]
It seems Grafana  is installing xorg-x11-server-utils. It seems X11 is installed as a [dependency](https://github.com/grafana/grafana-image-renderer/issues/159) for [grafana-image-renderer](https://github.com/grafana/grafana-image-renderer) plugin. But we're not using image renderer and it's not installed, but we have X11 packages. I want to know what in Grafana is using this X11 package and if it can be removable. Thanks in advance!
```
$ rpm -qa xorg-x11-server-*
xorg-x11-server-utils-7.7-27.el8.x86_64
```

**Comment by gelicia** [comment]
I dont see this dependency in our repo anymore. If there's something I'm missing, feel free to create an [issue](https://github.com/grafana/grafana/issues) about it.
